digitalstrategie/assets/scss/components/_richtext.scss

```diff
@@ -18,4 +18,9 @@
     .richtext-image.left {
         max-width: 65%;
     }
+}
+
+// needed for bullet point cut off on safari
+.textile ul > li {
+    margin-left: 1.5px;
```

Comment:
they are still very slightly cut off for me on safari / browserstack

Reply:
what version